yukyungdeok/10952.py

```diff
@@ -0,0 +1,7 @@
+# 10952번 A+B - 5
+while True:
+    a,b = map(int, input().split())
+    if (a == 0 and b == 0):
+        break;
+    else:
```

Comment:
```suggestion
print(a+b)
```
else 문이 따로 필요없으면 그냥 내려서 출력하는게 한 줄 줄일 수 있어